Classes/UT3HellbenderRearGunPawn.uc

```diff
@@ -39,10 +40,96 @@
 
 class UT3HellbenderRearGunPawn extends ONSPRVRearGunPawn;
 
+Var rotator SpineDrive;
+Var rotator ThighDriveL,ThighDriveR;
+Var rotator CalfDriveL,CalfDriveR;
+Var rotator FootDriveL,FootDriveR;
+Var rotator ArmDriveL,ArmDriveR;
+Var rotator ForeArmDriveL, ForeArmDriveR;
+Var rotator NeckDrive;
+var bool bAttachedDriver;
+
+simulated function AttachDriver(Pawn P)
+{
+    super.AttachDriver(P);
+    bAttachedDriver=true;
+
+    NeckDrive.Yaw=4000;
```

Comment:
...remove these definitions and put them into `defaultproperties` instead.

Reply:
Doesn't work, it won't compile that way

Reply:
It will as long as it is in the defaultproperties rotator form of NeckDrive=(Yaw=4000) etc.